MerlinAU.sh

```diff
@@ -6377,31 +6402,16 @@ Please manually update to version ${GRNct}${MinSupportedFirmwareVers}${NOct} or
     "$inMenuMode" && _WaitForEnterKey_ "$theMenuReturnPromptMsg"
 }
 
-##-------------------------------------##
-## Added by Martinski W. [2024-Nov-12] ##
-##-------------------------------------##
-_WAN_IsConnected_()
-{
-   local retCode=1
-   for iFaceNum in 0 1
-   do
-       if [ "$(nvram get wan${iFaceNum}_primary)" -eq 1 ] && \
-          [ "$(nvram get wan${iFaceNum}_state_t)" -eq 2 ]
-       then retCode=0 ; break ; fi
-   done
-   return "$retCode"
-}
-
-##----------------------------------------##
-## Modified by Martinski W. [2024-Nov-13] ##
-##----------------------------------------##
+##------------------------------------------##
+## Modified by ExtremeFiretop [2024-Nov-16] ##
+##------------------------------------------##
 _PostUpdateEmailNotification_()
 {
    _DelPostUpdateEmailNotifyScriptHook_
    Update_Custom_Settings FW_New_Update_Changelog_Approval TBD
 
    local theWaitDelaySecs=10
-   local maxWaitDelaySecs=600  #10 minutes#
+   local maxWaitDelaySecs=480  #8 minutes#
```

Comment:
Dropped the wait back down to 8 minutes (double) what it should usually take.
Should be good enough for most connection types and routers.